src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < 3; i++) {
+            System.out.println(figureSupplier.getRandomFigure().infoFromFigure());
```

Comment:
The `getRandomFigure()` method might return `null` if the default case in the switch statement of `FigureSupplier` is reached. This can lead to a `NullPointerException` when calling `infoFromFigure()`. Ensure that `getRandomFigure()` never returns `null` or handle the `null` case appropriately.